docusaurus/docs/operate/morse_migration/4_state_transfer_playbook.md

```diff
@@ -36,32 +36,36 @@ mkdir -p $HOME/morse-snapshot
 # 1. Untar the snapshot file
 tar -xvf <snapshot-file>.tar -C $HOME/morse-snapshot
 # 2. Change directory to the extracted snapshot folder
-$HOME/morse-snapshot
-# 3. Create the data directory
-mkdir data
-# 4. Move all *.db files to the data directory
-mv *.db data
+cd $HOME/morse-snapshot
 ```
 
-**⚠️ Note the height and date of the snapshot⚠️**
+:::warning Note the height and date of the snapshot
+The height and date are encoded in the snapshot file name.
+
+For example, the snapshot file name `pruned-166819-166919-2025-04-29.tar` has a max height of `166918` (NOTE: the end-height is exclusive) and a date of `2025-04-29`.
+:::
 
 ## 2. Export Morse Snapshot State
 
 Choose the snapshot height, which must be less than or equal to the snapshot height retrieved above. **This will be the published canonical export height.**
 
 ```bash
-pocket util export-genesis-for-reset <HEIGHT> pocket --datadir $HOME/morse-snapshot > morse_state_export.json
+export SNAPSHOT_HEIGHT="<HEIGHT>"
+export SNAPSHOT_DATE="<DATE>"
```

Comment:
For dates, this will be useful w.r.t formatting.
```suggestion
export SNAPSHOT_DATE="<DATE>" # E.g. ...
```

Reply:
Do you feel strongly that the very concrete example called out in the warning in the section above is not sufficient?

I'm reluctant to add shell comments because they make zsh throw a fit == bad copy/paste experience.

Reply:
Seems okay!

They'll need to copy+pasta+edit anyhow.

![Screenshot 2025-05-06 at 12 44 43 PM](https://github.com/user-attachments/assets/029103ce-aa23-4ea2-8362-360198daa061)

Reply:
I'm not sure what I'm seeing is about then - thought it was zsh :thinking: 

![image](https://github.com/user-attachments/assets/d3d3c653-12e4-4f8c-a4ec-11f7e0027b02)